docs/contributor/unsorted/debugging-races.md

```diff
@@ -1,9 +1,9 @@
 (debugging-races)=
 <!-- TODO(gfouillet): do not merge into 4.0, or delete whenever merged (reason: related to mongodb) -->
 # Debugging races
-## Triggering races
+## Triggering unit test races
 
-Someone pointed me to the [stress test script](https://github.com/juju/juju/wiki/Stress-Test) on this wiki. While it was
+Someone pointed me to the stress test script in `juju/scripts/unit-test/stress-race.bash`. While it was
```

Comment:
If we are interested, there are also tools like: https://github.com/ColinIanKing/stress-ng which provide stress on your host system while you run other things.

Reply:
I've added this in a note -- thanks! 

PS This doc has a lot of other issues, so I'm thinking of this PR merely as a first step in the right direction. For example, right now it feels like a collection of tips more than a how-to and we'll probably want to turn it into a proper how-to at some point.